pymc3/tests/test_sampling.py

```diff
@@ -705,29 +705,53 @@ def test_variable_type(self):
 
 class TestSamplePPCW(SeededTest):
     def test_sample_posterior_predictive_w(self):
-        data0 = np.random.normal(0, 1, size=500)
+        data0 = np.random.normal(0, 1, size=50)
+        warning_msg = "The number of samples is too small to check convergence reliably"
 
         with pm.Model() as model_0:
             mu = pm.Normal("mu", mu=0, sigma=1)
             y = pm.Normal("y", mu=mu, sigma=1, observed=data0)
-            trace_0 = pm.sample()
+            with pytest.warns(UserWarning, match=warning_msg):
```

Comment:
these tests just check the shapes of returned elements, so just speeding them up a little